### ISSUE TITLE
Using threshold on logits instead of probabilities for keypoint visualization

### ISSUE THREAD

**Issue by kampta** [facebookresearch/Detectron#249]
It's not an issue but in line https://github.com/facebookresearch/Detectron/blob/master/lib/utils/vis.py#L140 for visualizing keypoints, `kp_thresh=2` is used on the keypoints mask output. My question is regarding why and when to threshold on logits instead of probabilities values in the next layer (which are used for computing loss).

What are typical heuristics or process to come to such a number in case I am training my own model?

(Please redirect me to appropriate forum if this is not the correct one)

**Comment by kampta** [comment]
@rbgirshick : any pointers? :)
